package/kernel/ksmbd/Makefile

```diff
@@ -0,0 +1,74 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ksmbd
+PKG_VERSION:=3.4.3
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/cifsd-team/cifsd/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=a910c55d9e6924775e00504eddd00b49788603af29d0772cb9fb6722c189f628
+
+PKG_MAINTAINER:=Andy Walsh <andy.walsh44+github@gmail.com>
```

Comment:
Hmm, should we burden upstream projects with possible openwrt-related requests or not?
(this has just one patch. so possibly?)

Reply:
I'd remove it. No other core package that I'm aware of has non-team maintainers.